From 9f587d9d645d1f0f53f1e5145062484d1d974584 Mon Sep 17 00:00:00 2001
From: Moxiu <moxiu3310ud4@proton.me>
Date: Sat, 11 Oct 2025 00:38:40 +0200
Subject: [PATCH] modified genfs_contexts to work with axionAOSP

---
 sepolicy/vendor/genfs_contexts | 6 ++++++
 1 file changed, 6 insertions(+)

diff --git a/sepolicy/vendor/genfs_contexts b/sepolicy/vendor/genfs_contexts
index 8668584..4c0dc67 100644
--- a/sepolicy/vendor/genfs_contexts
+++ b/sepolicy/vendor/genfs_contexts
@@ -45,3 +45,9 @@ genfscon sysfs /kernel/ems/emstune/aio_tuner
 
 # Module
 genfscon sysfs /module/scsc_bt/parameters/bluetooth_address                                                    u:object_r:sysfs_bt_writable:s0
+
+# stuff
+genfscon proc /sys/vm/dirty_writeback_centisecs     u:object_r:proc_dirty:s0
+genfscon proc /sys/vm/vfs_cache_pressure            u:object_r:proc_drop_caches:s0
+genfscon proc /sys/vm/dirty_ratio                   u:object_r:proc_dirty:s0
+genfscon proc /sys/kernel/sched_migration_cost_ns   u:object_r:proc_sched:s0
-- 
2.51.0

